src/renderer/features/balances/model/balance-subscription-model.ts

```diff
@@ -81,28 +105,39 @@ const unsubscribeBalancesFx = createEffect(async ({ subscription }: UnsubscribeP
     return;
   }
 
-  const [balanceUnsubs, lockUnsubs] = await subscription.subscription;
+  const [balanceUnsubs, lockUnsubs] = await Promise.all(subscription.subscription);
 
   balanceUnsubs.forEach((fn) => fn());
   lockUnsubs.forEach((fn) => fn());
 });
 
+const populateBalancesFx = createEffect((accounts: AccountId[]): Promise<Balance[]> => {
+  return balanceService.getBalances(accounts);
+});
+
+sample({
+  clock: walletModel.$activeAccounts,
```

Comment:
With this change we store balances only for active accounts, right?
What if we select multisig wallet? We also need to retrieve signatories' balances